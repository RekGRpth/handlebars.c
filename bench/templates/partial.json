{
    "peeps": [
        {
            "name": "Moe",
            "count": 15
        },
        {
            "name": "Larry",
            "count": 5
        },
        {
            "name": "Curly",
            "count": 1
        }
    ]
}