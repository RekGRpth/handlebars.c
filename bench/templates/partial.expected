Hello Moe! You have 15 new messages.Hello Larry! You have 5 new messages.Hello Curly! You have 1 new messages.
